sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/core/PropertyPaths.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+Copyright (c) 2019 Eclipse RDF4J contributors.
```

Comment:
This looks malformed, it's missing the leading asterisk.

Reply:
Huh. I just noticed that this consistently done wrong in the sparqlbuilder classes. No biggie. But I'll fix it.